test/test_testing.py

```diff
@@ -6161,6 +6161,28 @@ def test_user_secret_permissions(self, request: pytest.FixtureRequest):
         with pytest.raises(RuntimeError):
             secret.remove_all_revisions()
 
+    def test_secret_id_variants(self, request: pytest.FixtureRequest):
```

Comment:
Or, if this form is kept, maybe add a blank line here and there in the function body?

Reply:
I can split it into multiple tests, sure.

Reply:
Hmm, but then it ends up with a lot more duplication than I would like, particularly since all these secrets ought to behave in exactly the same way anyway. I'll just add some blank lines.